site/pages/react/how-to-set-up-smart-account-client.mdx

```diff
@@ -0,0 +1,53 @@
+
+const { client } = useSmartAccountClient({
+  type: "ModularAccountV2",
+  policyId: "", // replace with your gas manager policy ID (https://dashboard.alchemy.com/gas-manager) - create policy https://accountkit.alchemy.com/react/sponsor-gas#create-a-gas-manager-policy
+});
+```
+
+### Client customization
+
+You can customize your client by passing in the following parameters (all these parameters are optional):
+
+| parameter           |                                            type |                                                                                                                                                                                                                                                                                 description |
+| :------------------ | ----------------------------------------------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+| `type`              |                         `SupportedAccountTypes` |                                                                                                                                                                                        options are "ModularAccountV2", "LightAccount", "MultiOwnerLightAccount", "MultiOwnerModularAccount" |
+| `policyId`          |                                        `string` |                                                                                                                                                                                                       gas manager policy ID from the [Dashboard](https://dashboard.alchemy.com/gas-manager) |
+| `useSimulation`     |                                       `boolean` |                                                                                                                                                                      enables use of [`alchemyUserOperationSimulator`](/reference/account-kit/infra/functions/alchemyUserOperationSimulator) |
+| `customMiddleware`  |                            `ClientMiddlewareFn` |                                                                                                                                                                                                                                      custom middleware to run before requesting sponsorship |
+| `feeEstimator`      |                            `ClientMiddlewareFn` |                                                                                 function that estimates the fees for a user operation (if you are using our RPCs, it's important to use the [`alchemyFeeEstimator`](/reference/account-kit/infra/functions/alchemyFeeEstimator) middleware) |
+| `gasEstimator`      |
```

Comment:
intentionally have no description for `gasEstimator`?

Reply:
not intentional, good catch!